### DISCUSSION TITLE
Does this version of i3 supports for i3bar transparency?

### DISCUSSION THREAD

**Discussion by KierPrev** [i3/i3#5114] [Q&A]
I did put the` i3bar_command i3bar -t` under 70_bar, but values of opacity only give a black background.

**Comment by Airblader** [comment]
You need to run a compositor.

**Reply by KierPrev** [reply]
Of course. For example, picom.
It does make the windows transparent, but not the bar itself

**Comment by ankurdhama** [comment]
Add a opacity rule to picom config for the i3bar class.
For ex: (passing opactiy rule to picom from command line): `picom --opacity-rule='50:class_g="i3bar"'`

**Reply by KierPrev** [reply]
This works, but the text also goes transparent.
